src/borg/archive.py

```diff
@@ -21,10 +21,11 @@
 from .chunker import Chunker
 from .compress import Compressor
 from .constants import *  # NOQA
+from .crypto import IntegrityError as IntegrityErrorBase
```

Comment:
At the time I introduced ...Base, I was having troubles with the 2 different IntegrityError classes and this was an easy way to solve it.